<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="h_splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Keywords</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pb_add">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>add a keyword substitution</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>plus.bmp</normaloff>plus.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_delete">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>delete the selected keyword substitution</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>delete.bmp</normaloff>delete.bmp</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="v_splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListWidget" name="listWidget">
          <property name="toolTip">
           <string>key word</string>
          </property>
          <property name="styleSheet">
           <string notr="true">b</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="toolTip">
           <string>value of key</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Template</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSplitter" name="splitter3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPlainTextEdit" name="template">
          <property name="toolTip">
           <string>email template, edit this</string>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="mail_merge">
          <property name="toolTip">
           <string>sample of merged document</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #f8f8e4</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
